src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/SwaggerUI.cs

```diff
@@ -89,7 +89,7 @@ public ISwaggerUI AddServer(IHttpRequestDataObject req, string routePrefix, IOpe
 
             this._baseUrl = servers.First().Url;
 
-            absolutePath = new Uri(this._baseUrl).AbsolutePath.TrimEnd('/');
+            absolutePath = new Uri(this._baseUrl, UriKind.RelativeOrAbsolute).AbsolutePath.TrimEnd('/');
```

Comment:
I cloned your repo and run it on my local machine. Unfortunately, this doesn't render the Swagger UI properly. Would you please take a further look?

![image](https://user-images.githubusercontent.com/1538528/147091139-ae800771-7a66-4879-a353-1166c3f78435.png)

Reply:
Thanks for catching this! I just took a closer look at the code, maybe we just don't need to parse that URL? I have updated the code, could you take a look?